CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 3.0.0 (2022-03-15)
+- Add support for enhanced user activity monitoring.
+	- Introduced custom(name:) method
```

Comment:
@aalekz should be `screen`?

Reply:
There are two different versions of custom, either just the name or name and properties. But screen(name:) should also be added.

Reply:
Added `screen(name:)` to the changelog now.